modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -7,27 +7,55 @@
 
 You can define requirements for Pod resources such as CPU, memory, and storage. Redpanda Data recommends that you determine and set these values before deploying the cluster, but you can also update the values on a running cluster.
 
-NOTE: In a running Redpanda cluster, you cannot decrease the number of CPU cores. You can only increase the number of CPU cores.
+== Prerequisites
 
-To see the available resources on the worker nodes that you have provisioned:
+- See xref:deploy:deployment-option/self-hosted/kubernetes/k-requirements.adoc[] for the minimum worker node, memory, CPU, and storage requirements.
 
+- Make sure that your physical or virtual machines have enough resources to give to Redpanda. To see the available resources on the worker nodes that you have provisioned:
++
 [,bash]
 ----
 kubectl describe nodes
 ----
 
-== Prerequisites
+== Production considerations
+
+- <<cpu, Enable CPU pinning>>. This configuration allows Redpanda to maintain exclusive access to designated CPU cores, improving throughput and reducing latency caused by CPU context switches.
+
+- <<qos, Enable the `Guaranteed` quality of service class for Pods that run Redpanda>>. This setup ensures that the CPU and memory allocated to Redpanda are not subject to throttling or other contention issues, providing a stable and predictable performance environment.
+
+- <<memory, Enable memory locking>>. This configuration prevents the operating system from paging out Redpanda's memory to disk, which can significantly impact performance.
+
+== Limitations
 
-See xref:deploy:deployment-option/self-hosted/kubernetes/k-requirements.adoc[] for memory, CPU, and storage.
+You cannot decrease the number of CPU cores in a running cluster.
```

Comment:
Is there a piece of documentation explaining this a bit further? I understand the limitation but prior to fully understanding it I immediately think "That's some bullshit or your handwaving around a detail that's critically important for me to know".

Reply:
The only thing I could find was [this issue](https://github.com/redpanda-data/redpanda/issues/3668) where @dotnwat says that we don't allow brokers to restart with a reduced number of cores.

Reply:
Dang, yeah that's probably the best thing we've got for now. It's not a great reference but I think it would be useful for users to understand this is a limitation of redpanda/seastar not anything k8s adjacent.

Reply:
I will clarify that this is a Redpanda limitation 👍